corehq/apps/userreports/reports/util.py

```diff
@@ -69,17 +70,16 @@ def header_rows(self):
         ]]
 
     @memoized
-    def get_data(self):
-        return list(self.data_source.get_data())
+    def get_data(self, limit=None):
+        return list(self.data_source.get_data(limit=limit))
 
     @property
     @memoized
     def total_rows(self):
         return [self.data_source.get_total_row()] if self.data_source.has_total_row else []
 
-    @property
```

Comment:
Verified that the only call to `data_rows` is the one that's updated in this PR.